spec/README.md

```diff
@@ -1,10 +1,9 @@
 ## Configuration
 
-The specs are configured to run against `http://localhost:7474` by default. In order to change this in your local setup you can set the `NEO4J_URL` environment variable on your system to suit your needs.
+The specs are configured to run against `bolt://localhost:6998` by default. In order to change this in your local setup you can set the `NEO4J_URL` environment variable on your system to suit your needs.
 
 To make this easier, the neo4j spec suite allows you to add a `.env` file locally to configure this. For example, to set the `NEO4J_URL` you simply need to add a `.env` file that looks like this:
 
 ```
-NEO4J_URL=http://localhost:7475
+NEO4J_URL=bolt://localhost:7687
```

Comment:
The previous value did not for me, failing with
```
Neo4j::Driver::Exceptions::ClientException:
  Unsupported URI scheme: http
```

So, updated to a good default for bolt instead.